compiler/src/iree/compiler/GlobalOptimization/test/fuse_silu_horizontal_matmul.mlir

```diff
@@ -0,0 +1,73 @@
+// RUN: iree-opt --split-input-file --pass-pipeline="builtin.module(func.func(iree-global-opt-fuse-silu-horizontal-matmul,canonicalize))" %s | FileCheck %s --check-prefix=REASSOCIATE-CHECK
```

Comment:
We use `check-prefix` when we have multiple FileCheck with different flags. Can you remove `check-prefix` and just use `CHECK` in the lit test?